timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -436,6 +436,11 @@ def _fit(
             fine_tune_trainer_kwargs["disable_tqdm"] = fine_tune_trainer_kwargs.get("disable_tqdm", (verbosity < 3))
             fine_tune_trainer_kwargs["use_cpu"] = str(self.model_pipeline.inner_model.device) == "cpu"
 
+            if fine_tune_trainer_kwargs["use_cpu"]:
+                logger.warning(
```

Comment:
Think this is `logger.info` at best, and could also be documentation instead of a runtime warning.